### PR TITLE
Fix integration test failures

### LINKED ISSUE #105: Intermittent test failure for `TestREClientIntegration.test_admin_empty_query`

### PR DESCRIPTION
**Pull request by MrCreosote**
The tests were starting before the RE API was up. I hypothesize this explains
the 1 pass / 1 fail pattern - the first test would fail because the server
wasn't up, and the step that takes the server down wouldn't run. The next set
of tests would pass, and take the server down, and so on and so forth.

Closes #105 

- [n/a] I updated the README.md docs to reflect this change.

For changes to the codebase:

- [n/a] I have written tests to cover this change.
- [x] This is not a breaking API change OR
- [ ] This is a breaking API change and I have incremented the API version and added a summary to CHANGELOG.md.

### CODE REVIEW

**Review by n1mus — APPROVED** [review]
Nice. LGTM

File: README.md [hunk 1]
```diff
@@ -49,3 +49,10 @@ The CI service is available in the `relationapi` service
 * Merge from Develop to Main
 * Draft a release using semantic versioning (e.g 1.2.3) by creating a tag with the target of Main
 * Ensure release notes are updated manually and/or by AutoGenerate Release Notes
+
```
Comment (n1mus) [inline README.md:52]: Wait why is this removed?

Comment (MrCreosote) [reply README.md:52]: Sorry, I don't understand - there's nothing removed here?

Comment (n1mus) [reply README.md:52]: Oh. JK

File: scripts/run_tests.sh [hunk 2]
```diff
@@ -16,8 +16,10 @@ sh /app/scripts/start_server.sh &
 coverage erase
 # spec validation
 python -m spec.validate
+# wait for the RE service to be up so integration tests can pass
+python -m relation_engine_server.utils.wait_for api
 # run importer/, relation_engine_server/, and spec/ tests
 coverage run --branch -m pytest
-# RE client tests
+# RE client tests - seemss like these tests were already run in the line above, why run them again?
```
Comment (ialarmedalien) [inline scripts/run_tests.sh:23]: typo - `seems`

Comment (MrCreosote) [reply scripts/run_tests.sh:23]: Got it fixed locally, it'll show up in my next PR
